client/starwhale/core/model/model.py

```diff
@@ -430,7 +433,8 @@ def list(
             )
         return rs, {}
 
-    def buildImpl(self, workdir: Path, **kw: t.Any) -> None:
+    def buildImpl(self, **kw: t.Any) -> None:
+        workdir = kw["workdir"]
```

Comment:
It is more clear that `workdir` in param than kw.get(), why remove this?

Reply:
for dataset sdk,  `workdir` is the not required argument.

Reply:
updated